docs/general/serializers.md

```diff
@@ -217,7 +217,15 @@ The object being serialized.
 
 #### #root
 
-PR please :)
+Resource root which is included in `JSON` adapter. As you can see at [Adapters Document](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/adapters.md), `Attribute` adapter (default) and `JSON API` adapter dose not include root at top level.
+By default, the resource root comes from the class name of the serialized class.
+
+There are some ways to specify root like below:
+* [Overriding the root key](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#overriding-the-root-key)
+* [Setting `type`](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/serializers.md#type)
+* Passing an argument like `root: 'specific_name'` to the serializer's initialization.
+
+You can also see examples of root at linked pages above.
```

Comment:
This means you can find some examples how 'root' is used in the pages linked above. I added this line because I thought it might be easier for reader if they can find some examples of usage. But now I guess maybe this is not really needed. So I will remove this line!